### ISSUE TITLE
`hot-formula-parser.d.ts` is not bundled with library

### ISSUE THREAD

**Issue by michaeloliverx** [iddan/react-spreadsheet#257]
Using this library in a typescript environment results in a type error at build time:
```
❯ npm run build
> tsc && vite build    

node_modules/react-spreadsheet/dist/index.d.ts:2:35 - error TS7016: Could not find a declaration file for module 'hot-formula-parser'. implicitly has an 'any' type.
  Try `npm i --save-dev @types/hot-formula-parser` if it exists or add a new declaration (.d.ts) file containing `declare module 'hot-formula-parser';`

2 import * as hotFormulaParser from 'hot-formula-parser'; 
                                    ~~~~~~~~~~~~~~~~~~~~

node_modules/react-spreadsheet/dist/index.d.ts:3:24 - error TS7016: Could not find a declaration file for module 'hot-formula-parser'. implicitly has an 'any' type.
  Try `npm i --save-dev @types/hot-formula-parser` if it exists or add a new declaration (.d.ts) file containing `declare module 'hot-formula-parser';`   

3 import { Parser } from 'hot-formula-parser';
                         ~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/react-spreadsheet/dist/index.d.ts:2
```

It looks like the [`hot-formula-parser.d.ts`](https://github.com/iddan/react-spreadsheet/blob/master/src/typings/hot-formula-parser.d.ts) file isn't being bundled:

![image](https://user-images.githubusercontent.com/55017335/195095304-47a8561c-5bf4-45ed-a0d4-db2e126ab3bb.png)

Manually adding this declaration file into my project fixes the build error for me, can this file be bundled?

**Comment by iddan** [comment]
Hey @michaeloliverx sorry for the late comment. I was on a vacation :) Would you like to open a PR to add it? I would love to review it

**Comment by iddan** [comment]
This is solved with `react-spreadsheet@0.8.0`
